src/components/Auth/index.js

```diff
@@ -38,7 +38,7 @@ const useAuth = () => {
   return state;
 };
 
-const AuthProvider = (props) => {
+function AuthProvider(props) {
```

Comment:
why did you change these to function?

Reply:
this was autofixed to conform with [this rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md) from the latest version of the react eslint plugin